### PR TITLE
release: update samples tag 1.5.0

### PR DESCRIPTION
**Pull request by varshaprasad96**
**Description of the change:**
release: update samples tag 1.5.0

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@ SHELL = /bin/bash
 # This value must be updated to the release tag of the most recent release, a change that must
 # occur in the release commit. IMAGE_VERSION will be removed once each subproject that uses this
 # version is moved to a separate repo and release process.
-export IMAGE_VERSION = v1.4.0
+export IMAGE_VERSION = v1.5.0
 # Build-time variables to inject into binaries
```
Comment (camilamacedo86) [inline Makefile:8]: /hold
Can we get merged a few docs before the release? 
e.g https://github.com/operator-framework/operator-sdk/pull/4640

Comment (camilamacedo86) [inline Makefile:8]: what about doing the release after Monday meeting and give a chance for the community to say if it has something that should be added before it rollout?

Comment (estroz) [reply Makefile:8]: This release has already been delayed for awhile. All issues in the v1.5.0 milestone have been completed, so there's no reason to block. Feel free to bring up your concerns in the next grooming meeting.

**Review by estroz — APPROVED** [review]
/lgtm

**Pull request merged**